### ISSUE TITLE
About the 'loss.step()' in Trainer.train(), trainer.py 

### ISSUE THREAD

**Issue by Vast-Stars** [yulunzhang/RNAN#1]
``` python
# in code/trainer.py
def train(self):
        self.scheduler.step()
        self.loss.step()
```

Is the `loss.step()` only available for those loss functions that contain parameters update operating such as the loss of GAN?

**Comment by yulunzhang** [comment]
Hi,
Sorry for my late reply. I'm not quite understand your problem. You can send me emails with more details. Thanks.
